.github/ISSUE_TEMPLATE/release.md

```diff
@@ -8,88 +8,98 @@ assignees: ''
 
 # Open Match Release Process
 
-Follow these instructions to create an Open Match release.  The output of the
-release process is new images and new configuration.
+Follow these instructions to create an Open Match release. The output of the
+release process are new images and new configuration.
 
 ## Getting setup
 
-*note: the commands below are pasted from the 0.5 release.  make the necessary
-changes to match your naming & environment.*
+**NOTE: The instructions below are NOT strictly copy-pastable and assume 0.5**
+**release. Please update the version number for your commands.**
 
 The Git flow for pushing a new release is similar to the development process
 but there are some small differences.
 
-**1. Clone your fork of the Open Match repository.**
+### 1. Clone Repository
 
 ```shell
+# Clone your fork of the Open Match repository.
 git clone git@github.com:afeddersen/open-match.git
```

Comment:
Why are we cloning from your branch, but then pushing to upstream?

Reply:
I don't know git very well. Honestly, I just created the branch from Github because it was just easier.

Do you have a recommendation here?

Reply:
Actually upstream is protected, this should be fine now.